src/sketch.js

```diff
@@ -212,8 +223,31 @@ function draw()
     // set background color
     background(...BACKGROUND_COLOR);
 
+    const step_params = {x: 0.2, y: 0.2, speed: 0.18}
+
+    if(soundOn)
```

Comment:
Please comment this test.

`condition evaluated: action taken`

Reply:
👍

Reply:
Use the standard format, in this case:

```suggestion
   // soundtrack is playing: change animation
    if(soundOn)
```